src/components/MediaList/LicenseBylineDescriptionList.tsx

```diff
@@ -18,6 +18,7 @@ const StyledList = styled("ul", {
   base: {
     listStyleType: "disc",
     listStylePosition: "inside",
+    marginBottom: "9",
```

Comment:
Bruk en semantic token. Designet skal egentlig kun innholde navngitte verdier, men det er ikke alltid sånn dessverre. I tilfeller som dette går vi bare for nærmeste ("small")

Reply:
Trodde hele poenget med det nye systemet var at vi kunne være mer fleksible 🙃

Reply:
Til en viss grad. Vi skal lene oss mer på designsystemet enn vi gjorde før hvertfall! Håpet er at vi får et litt mer konsistent look.